src/components/ControlBar.js

```diff
@@ -14,11 +14,17 @@ const controlBarStyle = {
   justifyContent: 'space-evenly',
 };
 
-const ConnectedControlBar = (props): React.MixedElement => {
+const ControlBar = (): React.MixedElement => {
```

Comment:
our team uses function declaration for top level, and function expression for every function inside a function